DNA Sequence Comparison Code.py

```diff
@@ -1,46 +1,67 @@
 from Bio import SeqIO
-from Bio import pairwise2
+from Bio.Align import PairwiseAligner
+from Bio.Seq import Seq
+
 multi_seqfn = "./data/project_dog_dna/dog_breeds.fa"
 mystery_seqfn = "./data/project_dog_dna/mystery.fa"
 
 def DNA_Seq_ID(multi_seqfn: str) -> dict:
     """ Reads in FASTA file and returns a dictionary of sequences.
 
+        Description
+
+        Parameters -
+        Returns - 
 
-        Example:"""
+        Example:
+    """
     sequences = {}
+
     with open(multi_seqfn, 'r') as f:
         for record in SeqIO.parse(f, "fasta"):
             sequences[record.id] = str(record.seq) #uses biopython to iterate over each line in the file and store the sequences in a dictionary
     return sequences
 
-def find_best_match(mystery_seqfn: str, sequences: dict) -> tuple:
+def find_best_match(mystery_seq: str, sequences: dict) -> tuple:
     """ Find the closest sequence in the multi-sequence file to the mystery sequence.
 
         Takes the dictionary from FASTA file conversion dictionary
+
+        Parameters - 
+        Returns - 
+
+        Example:
     """
+    Aligner = PairwiseAligner()
     best_match = None
     best_score = -1
     best_alignment = None
     best_seq_id = None
     
-    with open(multi_seqfn, 'r') as f:
-        mystery_seq = "".join([str(record.seq) for record in SeqIO.parse(f, "fasta")])
+    mystery_seq_object = seq(mystery_seq)
 
-        for seq_id, seq in sequences.items():
-            alignments = pairwise2.align.globalxx(mystery_seq, seq, score_only = True)
-            score = alignments
+    for seq_id, seq in sequences.items():
+        seq_obj = Seq(seq)
+        alignments = aligner.align(mystery_seq_object, seq_obj)
+        score = alignments[0].score
 
-            if score > best_score:
-                best_score = score
-                best_match = seq
-                best_seq_id = seq_id
-                best_alignment = pairwise2.align.globalxx(mystery_seq, seq)[0]
+        if score > best_score:
+            best_score = score
+            best_match = seq
+            best_seq_id = seq_id
+            best_alignment = Bio.Align.PairwiseAligner(mystery_seq, seq)[0]
```

Comment:
As score is computed from `alignements[0].score`, you probably want to use `alignements[0]` as object for the best alignment, provided `score > best_score`.

What you're doing currently is creating a new `PairwiseAligner` object, which you don't need, as you've already got `alignments[0]`.